README.md

```diff
@@ -1,3 +1,6 @@
+Notice
```

Comment:
Two suggetions: 

1. Move the new text between the `VeraDemo - Blab-a-Gag` and `About` lines
2. Format the Notice to make it stand out with:

```
| :information_source:  Notice |
|------------------------------|
| This project is ...          |
```
that should render like
| :information_source:  Notice |
|------------------------------|
| This project is ...          |